accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a project workspaces.
+
+# Motivation
+
+With the addition of the Workspace feature, it will become handy to be able to execute commands inside other workspaces than the one in the current directory.
+
+# Detailed design
+
+This RFC suggests to add the two following commands:
+
+```
+$> yarn workspace <project-name> pwd
+$> yarn workspace <project-name> <command-name> ...
```

Comment:
Hm, I see what you mean. The `pwd` command could be delegated to shell via `exec`. However, I think it would break the Windows portability. Since the command would run in cmd.exe, the pwd utility wouldn't be available, and users would have to use `cd` instead. In the end, the command would look like:

```
$> yarn workspace babel-core exec pwd || yarn workspace babel-core exec cd
```

Which looks awfully verbose (and not great perf-wise). Adding a builtin `pwd` command would solve this at virtually no cost, since we're need to compute the path anyway.

Regarding the `exec` command, I'm not sure how people will end up using it (they should probably register their commands as scripts if they intend to use them regularly, or to use advanced shell structures), but I have no objection implementing it, the maintenance cost shouldn't be very high anyway.

Just one concern: since Yarn already has a `run` command, aren't you worried that users might be confused regarding the command to use, depending on whether they want to run a script or a shell command?